libs/vm-modules/tests/unit/fixed_point.cpp

```diff
@@ -217,24 +217,25 @@ TEST_F(FixedPointTest, multidigit_integral_fixed_point64)
   EXPECT_TRUE(toolkit.Run());
 }
 
-TEST_F(FixedPointTest, multidigit_integral_fixed_point128)
-{
-  char const *TEXT = R"(
-    function main()
-      var val128_1 = 123fp128;
-      var val128_2 = toFixed128(123);
-      assert(val128_1 == val128_2);
-      var val128_3 = 123.456fp128;
-      var val128_4 = toFixed128(123.456);
-      var val32_1 = 123.456fp32;
-      var val64_1 = 123.456fp64;
-      assert(abs(val128_3 - toFixed128(val32_1)) < toFixed128(0.0003));
-      assert(abs(val128_3 - toFixed128(val64_1)) < toFixed128(0.00000012));
-    endfunction
-  )";
-
-  EXPECT_TRUE(toolkit.Compile(TEXT));
-  EXPECT_TRUE(toolkit.Run());
-}
+//???
```

Comment:
why is this commented out? it's not using float?

Reply:
Uncommented now; I was initially commenting everything out to save time